ivy/func_wrapper.py

```diff
@@ -1486,16 +1486,21 @@ def _handle_complex_input(
         jax_like = fn.jax_like if hasattr(fn, "jax_like") else "entire"
```

Comment:
a bit silly, but do we rename `handle_complex_input` to `handle_complex_inputs` 😅 ?

Reply:
It's only used for (and is only designed for) activation functions so I think the singular works better

Reply:
yeah you're right, `handle_complex_input` sounds better for it's purpose